app/views/signitem.html

```diff
@@ -66,92 +66,98 @@ <h4 class="col-xs-10">{{sc.fileName}}</h4>
         <div class="col-xs-12 container">
             <div class="row">
                 <div class="col-xs-2 col-sm-1">
-                    <i class="col-xs-12 col-sm-6 glyphicon glyphicon-home left" aria-hidden="true" ng-click="goHome()"></i>
-                    <i class="hidden-sm col-sm-6 glyphicon glyphicon-info-sign left" aria-hidden="true"></i>
-                </div>
-                <div class="col-xs-12 col-sm-11">
-                    <div class="col-xs-12 col-sm-2 btn-group">
-                        <button id="sc.btnModel.doc.id" type="button" class="btn btn-block db-btn-prim ng-class: {active: sc.isActive(sc.btnModel.doc.id)}"
-                        ng-click="sc.actionDoc()" ng-model="sc.btnModel.doc.active" ng-disabled="sc.isDisabled(sc.btnModel.doc.id)">
-                            <span translate translate-cloak>STR_SIGNING_REQ</span>
-                            <div>
-                                <span ng-hide="!sc.displayStatus" class="label ng-class:[sc.statusStyle(sc.item.Status)]" translate translate-cloak>{{sc.statusStrId(sc.item.Status)}}</span>
-                            </div>
+                    <div class="col-xs-6">
+                        <button type="button" class="btn btn-xs db-btn-hdr glyphicon glyphicon-home center" ng-click="sc.goHome()">
```

Comment:
Yes, but this way controller deactivates the buttons and activates progress bar before state.go jus in case it takes a moment.